systemtest/src/test/java/io/strimzi/systemtest/olm/OlmAbstractST.java

```diff
@@ -29,14 +33,32 @@
 
 import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
 
+@SuiteDoc(
+    description = @Desc("Provides an abstract base for OLM (Operator Lifecycle Manager) system tests, " +
+                        "offering common methods for deploying and verifying Strimzi custom resources " +
+                        "within an OLM-managed environment."),
+    labels = {
+        @Label(TestDocsLabels.OLM)
+    }
+)
```

Comment:
Do we really want this for abstract classes? It seems that final md doesn't contain much info about tests that are extended by it.

Reply:
I mean, it doesn't hurt if there is such info, I think. WDYT @strimzi/system-test-contributors ?

Reply:
> Do we really want this for abstract classes? It seems that final md doesn't contain much info about tests that are extended by it.

And maybe adding such a feature to `test-docs-generator-maven-plugin` wouldn't be bad.

Reply:
FMPOV I would use the documentation for the tests really and not what the abstract classes do - because we don't have any documentation about the `AbstractST` for example. If it would add some value like you describe the testcases there, then fine I guess. But in this case, you can have the common Javadoc comment.

Reply:
Yeah I think that it doesn't hurt as well, but since there are no test cases I think it is not of much value.

Reply:
Update to have simple javadoc instead of annotations :) Thanks.